src/web/components/global-nav/global-nav.scss

```diff
@@ -24,4 +24,6 @@
 }
 .hig__global-nav--open .hig__global-nav__container{
     margin-left: $hig__global-nav__sidenav--width;
-}
\ No newline at end of file
+}
+
+.hig--hidden { display: none; }
```

Comment:
When we use SMACSS/BEM to organize our CSS we want to have one file per module.  All CSS for that module should be in that file and nothing else.

E.g. This file `global-nav.scss` should only contain styles for the `global-nav` module, and every selector in the file should begin with `hig__global-nav`. 

For a general-purpose state style like this I would add it to a new file `src/web/helpers/css/states.scss`. Don't let it block you though. I'll move it with my PR.